### DISCUSSION TITLE
Github Pages picking up unknown domain

### DISCUSSION THREAD

**Discussion by harshal2030** [community/community#142036] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I'm trying to deploy jekyll site on github pages and idk but it's getting deployed to the testreactapp.me, which can't be accessed. Also, the harshal2030.github.io doesn't seems to be working.

Below is the image
<img width="883" alt="Screenshot 2024-10-20 at 11 25 11 AM" src="https://github.com/user-attachments/assets/04fdacda-7d9e-43fe-915d-b39a1331fee1">

Is there any specific area, I need to look at?

**Comment by rijonshahariar** [comment]
![image](https://github.com/user-attachments/assets/06c1f386-a33a-4d4d-a397-517905c45529)

Remove the CNAME and Try again
